url-gen/src/test/kotlin/com/cloudinary/transformation/ResizeTest.kt

```diff
@@ -163,6 +163,10 @@ class ResizeTest {
 
     @Test
     fun testFill() {
+        cldAssert("c_fill,x_100,y_100", Resize.fill { offsetX(100)
```

Comment:
I'm not sure this does anything, however the test is correct.

It looks like it needs to be appended to g_xy_center

See this 
https://res.cloudinary.com/demo/image/upload/h_2400,w_1080,c_fill,g_xy_center,x_500/sample.jpg

vs this

https://res.cloudinary.com/demo/image/upload/h_2400,w_1080,c_fill,x_500/sample.jpg